strapi/strapi
# fix: typescript autogeneration on develop command

### What does it do?

Adds the missing typescript build process with errors ignored to generate types before generating again with errors on.

### Why is it needed?

[ts autogeneration is broken ](https://github.com/strapi/strapi/issues/18738)

### How to test it?

In a typescript project, create a new content type in the content type builder -> it should now properly generate types and restart the server, unless there are legitimate typescript errors

### Related issue(s)/PR(s)

Fixes https://github.com/strapi/strapi/issues/18738
